e2e/portalicious/tests/ExportPeopleAffectedList/ExportDuplicateRegistrations.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test } from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { registrationsPV } from '@121-service/test/registrations/pagination/pagination-data';
+
+import BasePage from '@121-e2e/portalicious/pages/BasePage';
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import RegistrationsPage from '@121-e2e/portalicious/pages/RegistrationsPage';
+import TableComponent from '@121-e2e/portalicious/pages/TableComponent';
+
+// Export duplicate registrations
+const id = 2;
+const status = 'included';
+const fsp = 'Albert Heijn voucher WhatsApp';
+const name = 'Jan Janssen';
+const duplicateWithIds = '3';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+  const programIdPV = 2;
+  const pvProgramId = programIdPV;
```

Comment:
Why are these extra steps necessary here? (and in all other repetitions in this PR?)

We already have some mock-helper or other entity that we import everywhere that contains this very specific program-id?

Reply:
It is a strange habit from the past. Fixing it.